underfs/cephfs-hadoop/src/main/java/alluxio/underfs/cephfshadoop/CephfsHadoopUnderFileSystemFactory.java

```diff
@@ -11,21 +11,31 @@
 
 package alluxio.underfs.cephfshadoop;
 
+import alluxio.AlluxioURI;
 import alluxio.Constants;
+import alluxio.underfs.UnderFileSystem;
 import alluxio.underfs.UnderFileSystemConfiguration;
-import alluxio.underfs.hdfs.HdfsUnderFileSystem;
 import alluxio.underfs.hdfs.HdfsUnderFileSystemFactory;
 
+import com.google.common.base.Preconditions;
+
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Factory for creating {@link HdfsUnderFileSystem}.
+ * Factory for creating {@link CephfsHadoopUnderFileSystem}.
  *
- * It caches created {@link HdfsUnderFileSystem}s, using the scheme and authority pair as the key.
+ * It caches created {@link CephfsHadoopUnderFileSystem}s,
+ * using the scheme and authority pair as the key.
```

Comment:
This is no longer the case. Please remove the misleading comment, and I'll create another PR cleaning up the comment inherited from HDFS UFS factory.

Reply:
Remove it.